packages/matter.js/src/fabric/FabricManager.ts

```diff
@@ -26,36 +26,55 @@ export class FabricManager {
         this.nextFabricIndex = this.fabricStorage.get("nextFabricIndex", this.nextFabricIndex);
     }
 
+    getNextFabricIndex() {
+        const startingIndex = this.nextFabricIndex;
+        let iteration = 0;
+        while (true) {
```

Comment:
Logic looks correct AFAICT but I'd maybe write as
 
    for (i = 0; i < 255; i++) {
        fabricIndex = (this.nextFabricIndex + i) % 256;
        ...

Just to guarantee no possibility of infinite loop

Reply:
good idea

Reply:
adjusteds, but a bit different toi still remember if we gave an ID out which was not revoked